content/docs/modes/arp.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "ARP"
+weight: 21
+description: >
+  kube-vip ARP mode
+---
+
+ARP (sometimes called layer 2) is a simplistic protocol that is used to update the underlying network that in order to reach a certain IP address traffic should be sent to a specific piece of hardware. ARP will typically broadcast updates to the entire network that update the IP to Hardware (MAC) mapping, so that a host knows which physical or virtual network nic should be used to send traffic to.
+
+### Control plane
+
+When using ARP with the control plane feature, then a `leaderElection` will take place in order to determine the leader. This leader will then host the control place IP address and use ARP (through a gratuitous ARP broadcast) to update the network accordingly.
```

Comment:
Avoid using code formatting for concepts. The concept/process here is leader election. In the Services section, this is activated via `leaderElection`.